src/main/java/net/snowflake/ingest/streaming/example/SnowflakeStreamingIngestExample.java

```diff
@@ -25,17 +23,14 @@
   private static String PROFILE_PATH = "profile.json";
   private static final ObjectMapper mapper = new ObjectMapper();
 
-  public static void main(String[] args)
-      throws ExecutionException, InterruptedException, IOException {
+  public static void main(String[] args) throws Exception {
     ObjectNode profile =
         (ObjectNode) mapper.readTree(new String(Files.readAllBytes(Paths.get(PROFILE_PATH))));
     Properties prop = Utils.getPropertiesFromJson(profile);
 
     // Create a streaming ingest client
-    SnowflakeStreamingIngestClient client =
-        SnowflakeStreamingIngestClientFactory.builder("CLIENT").setProperties(prop).build();
```

Comment:
Is this still possible? Or must it always be wrapped with `try`?

Reply:
Yes, you can still initialize that this, and call close explicitly when you finish

Reply:
Got in - in that case should we keep the close call above?

Reply:
hmmm, why? I add a try block so close will be automatically called?